end-to-end_sales-forecasting/sales_forecasting-end-to-end.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
```

Comment:
<p>Seems like some formatting off with this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='178'/>

Reply:
<p>Purposefully in raw format -- if you want to sample the data and produce FIs for &lt; len(prod data), then users can use that cell </p>